package/src/skia/web/JsiSkPaint.ts

```diff
@@ -66,6 +66,10 @@ export class JsiSkPaint extends HostObject<Paint, "Paint"> implements SkPaint {
     this.ref.setAntiAlias(aa);
   }
 
+  setDither(aa: boolean) {
+    // this.ref.setDither(aa);
```

Comment:
we should throw a warning here

Reply:
It's now available in SkiaWASM so we can keep it here.